index.html

```diff
@@ -1538,6 +1538,28 @@ <h4><dfn data-lt="version-time" data-dfn-type="dfn" id="version-time">version-ti
       </pre>
     </section>
 
+    <section>
+      <h4><dfn data-lt="relative-ref" data-dfn-type="dfn" id="relative-ref">relative-ref</dfn></h4>
```

Comment:
```suggestion
      <h4><dfn data-lt="path" data-dfn-type="dfn" id="path">path</dfn></h4>
```

Nitpicking the name... relative-ref is technically accurate, but there are two downsides to the name... 1) it requires understanding of what a relative reference is, which requires some understanding of the URI spec... which we shouldn't assume, and 2) it's hyphenated and harder to type than a single word.

Can we do something like `path` instead? It's not entirely technically accurate, but it does fit all of the use cases we can think of and it's easier for non-URI experts to understand what to do with the property. If we use `path`, there is also the benefit that when you read the URL, you will most likely read `service` and then `path`, which does give you an understanding of exactly what it is... it is the service path.

Reply:
Hmm `relative-ref` was originally proposed by @dlongley in https://github.com/w3c/did-core/issues/159#issuecomment-598286814. We already have a number of other hyphenated parameters (version-id, version-time, initial-state, transform-keys).

The problem I see with `path` is that it contains not only a path, but it can also contain query and fragment. Personally I would prefer the technically accurate name.

In any case, this PR only reflects was has already been added to DID Core in https://github.com/w3c/did-core/pull/357. Shouldn't we merge this first and then potentially open a new issue in DID Core to discuss renaming the parameter?

Reply:
> In any case, this PR only reflects was has already been added to DID Core in [w3c/did-core#357](https://github.com/w3c/did-core/pull/357). Shouldn't we merge this first and then potentially open a new issue in DID Core to discuss renaming the parameter?

Not if we can easily get it right the first time... if folks object to path and think relative-ref is better, then we can go ahead and merge. Just providing feedback on the PR.